types/pupa/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for pupa 1.0
+// Project: https://github.com/sindresorhus/pupa
+// Definitions by: nju33 <https://github.com/nju33>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function pupa(tpl: string, data: any[] | {[key: string]: any}): string | never;
```

Comment:
`string | never` is the same as `string`.  Please reduce it to `string`

Reply:
Fixed.
Thank you.